src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null || repeatPassword == null
+                || password.length() < 10 || repeatPassword.length() < 10
+                || !password.equals(repeatPassword)) {
```

Comment:
let's think if we need `repeatPassword == null` and `repeatPassword.length() < 10` conditions
`!password.equals(repeatPassword)` already covers these cases